.classpath

```diff
@@ -126,6 +126,5 @@
 			<attribute name="optional" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" path="/DEH-CommonJ"/>
```

Comment:
gone?

Reply:
Not gone but the reference from there was useless there since its reference as a maven dependency as well